### DISCUSSION TITLE
Facebook videos or custom videos

### DISCUSSION THREAD

**Discussion by stalkerGH** [welpo/tabi#389] [Q&A] [answered]
# Embededing Facebook or custom videos  

## Summary
I'd like to ask if it is possible to embed Facebook videos or custom, self-hosted videos like YT shortcode (example showed here: https://github.com/welpo/tabi/issues/371)

## Detailed Description
I have short MP4 movie (self-hosted file) which I'd like to embed on my Zola/tabi website. How can I do that?

**Comment by welpo** [comment, marked as answer]
The standard way to add videos (local files) is ([docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video)):

```html
<video src="filename.mp4" controls="controls"></video>
```

For a Facebook video, I assume their player provides an embed HTML code. Adding that to your page and allowing the necessary connections to Facebook through the [CSP](https://welpo.github.io/tabi/blog/security/) should work.

**Comment by stalkerGH** [comment]
Thank you!
